assets/base.css

```diff
@@ -3302,7 +3302,7 @@ details-disclosure > details {
 
   @keyframes slideIn {
     from {
-      transform: translateY(10vh);
+      transform: translateY(3rem);
```

Comment:
It's much better with this 👍 

I wonder if we should also reduce the speed at which the animation run 🤔 I think first visits are fine but then when you keep navigating, you don't want to have the user to start wondering weather it's the time it takes for things to load or if we're slowing down their experience for the sake of animation. 

What do you think ? I find `--duration-default` to be pretty decent.

Reply:
`--duration-default` felt a little too fast to me. I pushed a compromise in 128ad25 — let me know how that feels to you.

Reply:
Yeah it's looking nice 👌 Should be a good starting point and we can tune it more in the future if needed 👍

Reply:
Great! I've taken this out of draft and opened it up for reviews. 🌟